sdk/search/ai-search/README.md

```diff
@@ -0,0 +1,302 @@
+);
+```
+
+## Key concepts
+
+### SearchIndexClient
+
+`SearchIndexClient` is one of the primary interface for developers using the Cognitive Search client library. It provides asynchronous methods for working with documents in an index.
+
+## Examples
+
+### Query documents in an index
+
+To list all results of a particular query, you can use `listSearchResults` with a search string that uses [simple query syntax](https://docs.microsoft.com/en-us/azure/search/query-simple-syntax):
+
+```js
+for await (const result of client.listSearchResults("wifi -luxury")) {
+  console.log(result);
+}
+```
+
+For a more advanced search that uses [Lucene syntax](https://docs.microsoft.com/en-us/azure/search/query-lucene-syntax), specify `queryType` to be `all`:
+
+```js
+for await (const result of client.listSearchResults('category:budget AND "recently renovated"^3', {
+  queryType: "full",
+  searchMode: "all"
+})) {
+  console.log(result);
+}
+```
+
+### Querying with OData filters
+
+Using the `filter` query parameter allows you to query an index using the syntax of an [OData \$filter expression](https://docs.microsoft.com/en-us/azure/search/search-query-odata-filter).
+
+```js
+const { odata } = require("@azure/ai-search");
+
+let baseRateMax = 200;
+let ratingMin = 4;
+for await (const result of client.listSearchResults("WiFi", {
+  filter: odata`Rooms/any(room: room/BaseRate lt ${baseRateMax}) and Rating ge ${ratingMin}`,
+  orderBy: "Rating desc"
+  select: ["HotelId", "HotelName", "Rating"]
+})) {
+  // Each result will have "HotelId", "HotelName", and "Rating"
+  // in addition to the standard search result property "score"
+  console.log(result);
+}
+```
+
+### Querying with facets
+
+[Facets](https://docs.microsoft.com/en-us/azure/search/search-filters-facets) are used to help a user of your application refine a search along pre-configured dimensions. [Facet syntax](https://docs.microsoft.com/en-us/rest/api/searchservice/search-documents#facetstring-zero-or-more) provides the options to sort and bucket facet values.
+
+```js
+const page = (
+  await client
+    .listSearchResults("WiFi", {
+      facets: ["Category,count:3,sort:count", "Rooms/BaseRate,interval:100"]
+    })
+    .byPage()
+    .next()
+).value;
+
+// {
+//   Rating: [
+//     { count: 4, value: 2 },
+//     { count: 5, value: 3 },
+//     { count: 8, value: 4 }
+//   ],
+//   'Rooms/BaseRate': [
+//     { count: 16, value: 0 },
+//     { count: 17, value: 100 },
+//     { count: 17, value: 200 }
+//   ],
+//   Category: [
+//     { count: 5, value: 'Budget' },
+//     { count: 5, value: 'Luxury' },
+//     { count: 5, value: 'Resort and Spa' }
+//   ]
+// }
+console.log(page.facets);
+```
+
+When retrieving results, an `facets` property will be available that will indicate the number of results that fall into each facet bucket. This can be used to drive refinement (e.g. issuing a follow-up search that filters on the `Rating` being greater than 3 and less than 4.)
```

Comment:
```suggestion
When retrieving results, a `facets` property will be available that will indicate the number of results that fall into each facet bucket. This can be used to drive refinement (e.g. issuing a follow-up search that filters on the `Rating` being greater than 3 and less than 4.)
```

Reply:
BTW, your example of range is really small - "greater than 3 and less than 4". Did you mean 5? Or 3 inclusive?

Reply:
3 is inclusive, clarified, you're picking "3 star reviews"